examples/js/loaders/GLTFLoader.js

```diff
@@ -1897,8 +1897,6 @@ THREE.GLTFLoader = ( function () {
 
 					var attributeEntry = attributes[ attributeId ];
 
-					if ( attributeEntry === undefined ) return;
-
 					var bufferAttribute = dependencies.accessors[ attributeEntry ];
```

Comment:
I don't think this check is necessary because `undefined` entry, like the following, is very rare.

    "attributes": {
        "a": undefined
    }

Let me know if there's any reasons this line is necessary.